.editorconfig

```diff
@@ -0,0 +1,20 @@
+# EditorConfig is awesome: https://EditorConfig.org
+# If you are using VS Code consider using the following extension
+# https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{rst,js,py}]
+charset = utf-8
+
+[*.{rst,md}]
+indent_style = space
+indent_size = 2
+
+# Supported only by some editors
```

Comment:
Can you move this so that the max_line_length is next to "indent_size", so it looks to be part of the same group. The comment is useless in my opinion.

Reply:
I think the comment will help VS Code users (and maybe some other users) to understand why the `max_line_length` is not working as intended.

Reply:
All of the settings here might not work. Nothing says if the max line length should show an indicator, warning or automatically wrap to the next line. For example with Python, automatic wrapping long lines would most likely break the code.

Any case, you can remove the empty line or add the comment in the end of the line (if that was supported).